sql/driver/wire.go

```diff
@@ -80,14 +100,17 @@ func (d Datum) Value() (driver.Value, error) {
 		val = t.BytesVal
 	case *Datum_StringVal:
 		val = t.StringVal
+	case *Datum_DateVal:
+		val = Date{t.DateVal.GoTime()}
```

Comment:
Even with the time zone printed, it's surprising, is all I'm saying. It seems more consistent to let the display reflect the fact that the server does everything in UTC. But hopefully this all becomes moot soon enough when we teach the server about time zones.